samples/listTrainingPhrases.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(projectId, intentId, location, agentId) {
+  // [START dialogflow_list_training_phrases]
+  const {IntentsClient} = require('@google-cloud/dialogflow-cx');
+
+  // Create the intents client
+  const intentClient = new IntentsClient();
```

Comment:
Everything here needs to be wrapped in an inner async function so the code will work as is when copy/pasted in a node.js environment.  See https://g3doc.corp.google.com/company/teams/cloud-devrel/dpe/samples/nodejs.md?cl=head